siriuspy/siriuspy/search/id_search.py

```diff
@@ -0,0 +1,45 @@
+"""Search module."""
+
+
+class IDSearch:
+    """ID Search Class."""
+
+    _beamline2idname = {
```

Comment:
@anacso17, I think this should be a static table in control-system-constants, don't you?

Reply:
@fernandohds564 , I think it is a good idea. I'll merge this PR as it is, add a NOTE and move tha table to csconstants later.